CMakeLists.txt

```diff
@@ -190,7 +190,7 @@ install(FILES
   ${CMAKE_CURRENT_BINARY_DIR}/hrpsys-base.pc
   DESTINATION lib/pkgconfig)
 
-add_definitions(-DHRPSYS_PACKAGE_VERSION=\"\\"${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}\\"\")
+add_definitions(-DHRPSYS_PACKAGE_VERSION=\"${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}\")
```

Comment:
@kirohy what happens without this changes?

Reply:
Sorry, this change cannot built in Melodic. In Noetic wthout this change, hrpsys sample fails with `ValueError: invalid version number '"315.15.0"'`. This caued by python3 problem, so I'll revert this file and edit hrpsys python sample scripts.